aten/src/ATen/CMakeLists.txt

```diff
@@ -439,9 +439,10 @@ if(USE_CUDA AND NOT USE_ROCM)
   if($ENV{ATEN_STATIC_CUDA})
     list(APPEND ATen_CUDA_DEPENDENCY_LIBS
       ${CUDA_LIBRARIES}
-      CUDA::cusparse_static
```

Comment:
Why do you need to touch the other files too and why does the format need to change?